feat(object-storage): add doc on conditional writes (#6644)
scaleway/docs-content: pages/object-storage/api-cli/using-conditional-writes.mdx

---
title: Using conditional writes
description: Add logic to your requests and avoid accidental overwrites with conditional writes for Scaleway Object Storage.
tags: object storage object-storage conditional write overwrite etag if-match if-none-match header
dates:
  validation: 2026-07-03
  posted: 2026-07-03
---
import Requirements from '@macros/iam/requirements.mdx'

> Conditional writes allow you to add an HTTP header to your write requests to specify preconditions for Object Storage operations. The operation only proceeds if the precondition is met. Object Storage supports two conditional headers:

Reviewer: ```suggestion
Conditional writes allow you to add an HTTP header to your write requests to specify preconditions for Object Storage operations. The operation proceeds only if the preconditions are met. Object Storage supports two conditional headers:
```
